tests/sentry/test_event_manager.py

```diff
@@ -815,7 +815,7 @@ def test_tsdb(self):
             fingerprint=['totally unique super duper fingerprint'],
             environment='totally unique super duper environment',
         ))
-        event = manager.save(project)
+        event = manager.save(project.id)
```

Comment:
noidea.jpg, but it was failing for me

Reply:
As far as I can tell the call to `Project.objects.get_from_cache` used to be a cache hit and now it isn't anymore. In case of a cache miss it can't deal with passing a `Model`, in case of a cache hit it can.

We could just remove the logic that converts `Model` instances to their PKs since nobody can rely on it anyway.